packages/client/modules/userDashboard/components/UserTasksHeader/UserTasksHeader.tsx

```diff
@@ -98,7 +98,7 @@ const UserTasksHeader = (props: Props) => {
             user {
               id
               preferredName
-              tms
+              teamIds
```

Comment:
Here we now load all team members of the viewers team members, which isn't ideal.